app/src/main/java/ch/admin/bag/dp3t/networking/ConfigWorker.java

```diff
@@ -93,11 +93,16 @@ private static void loadConfig(Context context) throws IOException, ResponseErro
 
 		InfoBoxModel info = config.getInfoBox(context.getString(R.string.language_key));
 		if (info != null) {
-			secureStorage.setInfoboxTitle(info.getTitle());
-			secureStorage.setInfoboxText(info.getMsg());
-			secureStorage.setInfoboxLinkTitle(info.getUrlTitle());
-			secureStorage.setInfoboxLinkUrl(info.getUrl());
-			secureStorage.setHasInfobox(true);
+			if (info.getInfoId() == null || !secureStorage.getInfoboxId().equals(info.getInfoId())) {
```

Comment:
secureStorage.getInfoboxId() returns null as default and will lead to NPE, if you check info.getInfoId().equals(secureStorage.getInfoboxId()) you don't have this problem.